packages/devextreme/js/__internal/ui/chat/messagebox.ts

```diff
@@ -68,27 +84,52 @@ class MessageBox extends DOMComponent<MessageBox, Properties> {
     this._createMessageEnteredAction();
     this._createTypingStartAction();
     this._createTypingEndAction();
+    this._createMessageEditCanceledAction();
   }
 
   _initMarkup(): void {
-    $(this.element()).addClass(CHAT_MESSAGEBOX_CLASS);
+    $(this.element()).addClass(CHAT_MESSAGEBOX_WRAPPER_CLASS);
 
     super._initMarkup();
 
-    this._renderTextArea();
-    this._renderButton();
+    this._renderEditingPreview();
+    this._renderMessageBox();
+  }
+
+  _renderMessageBox(): void {
+    const $messageBox = $('<div>')
```

Comment:
Incorrect stracture: message box is area that contains field, button, preview. So, we need wrap content inside messagebox